test/integration_test.dart

```diff
@@ -418,6 +418,34 @@ defineTests() {
       });
     });
 
+    group('always_require_non_null_named_parameters', () {
+      IOSink currentOut = outSink;
+      CollectingSink collectingOut = new CollectingSink();
+      setUp(() {
+        exitCode = 0;
+        outSink = collectingOut;
+      });
+      tearDown(() {
+        collectingOut.buffer.clear();
+        outSink = currentOut;
+        exitCode = 0;
+      });
+
+      test('only throw errors', () {
+        dartlint.main([
+          'test/_data/always_require_non_null_named_parameters',
+          '--rules=always_require_non_null_named_parameters'
```

Comment:
Can we pass the flag in here?  If it doesn't already, can we make linter main pass arguments down to the analyzer?